Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/README.md

```diff
@@ -0,0 +1,268 @@
+SaaS Security is an integrated CASB (Cloud Access Security Broker) solution that helps Security teams like yours meet 
+the challenges of protecting the growing availability of sanctioned and unsanctioned SaaS applications and maintaining 
+compliance consistently in the cloud while stopping threats to sensitive information, users, and resources. 
+SaaS Security options include SaaS Security API (formerly Prisma SaaS) and the SaaS Security Inline add-on.
+
+
+## Configure SaaS Security on Cortex XSIAM
+
+1. Navigate to **Settings** > **Configurations** > **Data Collection** > **Automations & Feed Integrations**.
+2. Search for Saas Security Event Collector.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Your server URL | The instance configuration URL based on the server location. | True |
+    | Client ID | The SaaS Security Client ID. | True |
+    | Client Secret | The SaaS Security Secret ID. | True |
+    | Trust any certificate (not secure) | By default, SSL verification is enabled. If selected, the connection isn’t secure and all requests return an SSL error because the certificate cannot be verified. | False |
+    | Use system proxy settings | Uses the system proxy server to communicate with the  integration. If not selected, the integration will not use the system proxy server. | False |
+    | The maximum number of events per fetch. | How many events to fetch every time fetch is being executed. Default is 100. Must be divisible by 100 due to Saas-Security api limitations | True |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+## Create the Client ID and Client Secret on SaaS Security
+In the SaaS Security UI, do the following:
+1. Navigate to **Settings** > **External Service**.
+2. Click **Add API Client**.
+3. Specify a unique name for the API client.
+4. Authorize the API client for the required scopes. You use these scopes in the POST request to the /oauth/token endpoint. The Required Scopes are:
+    - Log access — Access log files. You can either provide the client log access API or add a syslog receiver.
+    - Incident management — Retrieve and change the incident status.
+    - Quarantine management — Quarantine assets and restore quarantined assets.
+6. Copy the client ID and client secret.
+Tip: Record your API client secret somewhere safe. For security purposes, it’s only shown when you create or reset the API client. If you lose your secret you must reset it, which removes access for any integrations that still use the previous secret.
+7. Add the **Client ID** and **Client Secret** to Cortex XSOAR.
+Note: For more information see the [SaaS Security Administrator's Guide](https://docs.paloaltonetworks.com/saas-security/saas-security-admin/saas-security-api/syslog-and-api-integration/api-client-integration/add-your-api-client-app.html)
+
+
+## Limitations
```

Comment:
Shouldnt this be the same list?

Reply:
what do you mean here?

Reply:
the same Limitations list as in 
Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/SaasSecurityEventCollector_description.md